src/aspire/commands/orient3d.py

```diff
@@ -29,7 +29,41 @@
     type=int,
     help="Max number of image rows to read from STAR file",
 )
-def orient3d(data_folder, starfile_in, starfile_out, pixel_size, max_rows):
+@click.option(
```

Comment:
I feel like as we bring in new command line tools or modify them we should perhaps bring them under simple testing just so we know with each change all the flags and so on will work. `click` comes packaged with a nice testing apparatus:

https://click.palletsprojects.com/en/8.0.x/testing/

I have an example usage in my PR:

https://github.com/ComputationalCryoEM/ASPIRE-Python/pull/520/files
(`tests/test_coordinate_source.py`)

Reply:
Added a command line test.